### DISCUSSION TITLE
mod not working

### DISCUSSION THREAD

**Discussion by Thedemon126** [Okabintaro/SubmersedVR#78] [Q&A]
i've been trying to play this mod for months. every time i open the game it starts in the original vr mode and i can't see my controllers.
i do have the advanced options on -vrmode openvr and the files are correctly extracted i've done the setup 3 times today just to make sure.
 i really want to play it and i would be really grateful if anybody helps
this is the logoutput:
[LogOutput.log](https://github.com/Okabintaro/SubmersedVR/files/12601352/LogOutput.log)
(Note - this game is pirated if that's the problem then i'll wait untill i buy it)

**Comment by Okabintaro** [comment]
The log shows you got an old unsupported version of the game. Please go and buy the game. It's awesome and you can get it pretty cheap on steam sales too.

**Reply by Thedemon126** [reply]
i did it with the newer version now i got this with the same problem.
[LogOutput.log](https://github.com/Okabintaro/SubmersedVR/files/12611276/LogOutput.log)
you're totally right this gmae deserves every penny and i'll buy it as sson as possible. i was playing on xbox and i wanted to try it on pc but i didn't have eough to buy it. and thank you for your time.

**Reply by Okabintaro** [reply]
This is still an outdated version that came before the steam deck update.
You might try an older version of the mod but please just get the steam version that is tested by everyone.
